tailscale/run.sh

```diff
@@ -6,7 +6,7 @@ set -eum
 CONFIG_PATH=/data/options.json
 TAILSCALE_SOCKET="/var/run/tailscale/tailscaled.sock"
 TAILSCALE_FLAGS=()
-TAILSCALED_FLAGS=("-state" "/data/tailscaled.state" "-socket" "$TAILSCALE_SOCKET")
+TAILSCALED_FLAGS=("-statedir" "/data" "-state" "/data/tailscaled.state" "-socket" "$TAILSCALE_SOCKET")
```

Comment:
Not sure if this is necessary. I received an error like `no CertDir` but that was before I set `--cert-file` and `--key-file` in the `tailscale cert` command. Maybe it was just necessary because Tailscale writes the certificates to the directory which is defined in `-statedir`.

Reply:
https://github.com/tailscale/tailscale/blob/3b3994f0db979674c2850872bf7acfaeafb9b69e/ipn/localapi/cert.go#L57